src/drivers/imx/sdma.c

```diff
@@ -691,6 +691,7 @@ static int sdma_prep_desc(struct dma_chan_data *channel,
 		 * For MEM2MEM the source is stored in buf_addr and destination
 		 * is in buf_xaddr.
 		 */
+		tr_dbg(&sdma_tr, "SDMA: elem index %d dir %d", i, config->direction);
```

Comment:
Before you remove this pull request from draft, please remove these prints. Debug prints can be added locally, however unless there's a _very_ clear case where they're useful they should generally not be submitted upstream.

Reply:
I would have removed them together with the whole sdma commit for a non-draft pull request. I just left it in here as a convenient way to ask my question about the watermark level.